packages/fxa-settings/src/components/LinkExpired/en.ftl

```diff
@@ -0,0 +1,12 @@
+## LinkExpired component
+
+# The user followed a password reset link, but that link is expired and no longer valid
+reset-pwd-link-expired-header = Reset password link expired
+
+# The user followed a password reset link, but that link is expired and no longer valid
+signin-link-expired-header = Confirmation link expired
+
+signin-link-expired-message = The link you clicked to confirm your email is expired.
+reset-pwd-link-expired-message = The link you clicked to reset your password is expired.
+# Button to request a new link to reset password or confirm email if the previous link was expired
+resend-link = Receive new link
```

Comment:
Is there a reason this needs a new ID? Since these have already been localized and string isn't changing the ID could be left as-is and existing translations could be used.

Reply:
Will do, thank you!